array.py

```diff
@@ -27,7 +27,7 @@ def f_contiguous_strides(itemsize, shape):
     if shape:
         strides = [itemsize]
         for s in shape[:-1]:
-            strides.append(strides[-1]*s)
+            strides.append(strides[-1]*max(1, s))
         return tuple(strides)
```

Comment:
There's an inlined copy of this here:

https://github.com/inducer/pyopencl/blob/fd60115e43673f809a0c0508571a67d195ab83dc/pyopencl/array.py#L529-L540

That also needs to be changed. (Inlined for "performance" reasons.)

Reply:
Got it! 

inducer/pyopencl#514